server/main.example.go

```diff
@@ -0,0 +1,26 @@
+package server
```

Comment:
Isn't ``bin`` conventionally the place where compiled binaries are placed? Maybe naming the package something like ``keyserver`` or ``coniksServer`` would be more descriptive.